client/js/monthData.js

```diff
@@ -1,151 +1,48 @@
 /*Temporary Mock Month Data */
```

Comment:
So if our server is down we still want the render an empty calendar, otherwise the page will just be empty. I havent figured out a great way to do this yet. The simplest way would be to render stub data like this.